contracts/contracts/strategies/README.md

```diff
@@ -42,19 +42,19 @@
 
 ![Convex ETH Metapool Strategy Storage](../../docs/ConvexEthMetaStrategyStorage.svg) -->
 
-## Generalized 4626 Strategy
+## Frax ETH Strategy
```

Comment:
nit: We still have the `Generalized4626Vault.sol` and might use that in future for other strategies, so, should we not remove those diagrams?

Reply:
I'm only generating diagrams for implementations as we'll have too many otherwise. `Generalized4626Vault` gets picked up in the hierarchy diagram and functions in the squashed diagram